whitepaper.md

```diff
@@ -0,0 +1,170 @@
+
+Currently Sawtooth does not dictate a privacy and confidentiality approach. The Sawtooth team has experimented with several approaches, including trusted computation of transaction off-chain, which can be found here - [https://sawtooth.hyperledger.org/docs/core/releases/0.8.8/examples/private_utxo/overview.html](https://sawtooth.hyperledger.org/docs/core/releases/0.8.8/examples/private_utxo/overview.html).
+
+This area is an active area of research and a key priority for future Sawtooth releases. The Sawtooth team believes that different privacy models are appropriate for different deployments and intends to leverage data isolation, cryptographic, and trusted hardware techniques in their approach.
+
+**Decentralized Governance**
+
+One of the unique features of Sawtooth is the on-chain governance mechanism enabled by the ‘settings’ transaction family. The settings transaction family provides a methodology for storing on-chain configuration settings such as which consensus mechanism is used and which transaction processors are enabled on the network
+
+This design of the settings transaction family supports two authorization options, either a single authorized key which can make changes or multiple authorized keys. In the case of multiple keys, a percentage of votes signed by the keys is required before a change is made.
+
+**Sawtooth and Ethereum Compatibility**
+
+The primary goal of the Sawtooth-Ethereum integration project, affectionately dubbed "Seth", is to add support for running Ethereum Virtual Machine smart contracts to the Hyperledger Sawtooth platform. In order to make this possible, the Hyperledger Sawtooth project worked with the [Hyperledger Burrow](https://github.com/hyperledger/burrow) project to integrate their EVM implementation, the Burrow EVM, into the Hyperledger Sawtooth platform.
+
+The secondary goal of Seth is to make it easy to port existing EVM smart contracts, and the DApps that depend on them, to Sawtooth. This has been largely accomplished by replicating the [Ethereum JSON RPC API](https://github.com/ethereum/wiki/wiki/JSON-RPC). This also enables developers to leverage existing Ethereum tooling such as IDEs when developing for Sawtooth.
+
+It is important to note that Seth is not a complete Ethereum implementation. The Sawtooth platform has made fundamental design decisions that differ from those made by the Ethereum platform. 
+
+**Conclusion**
+
+Sawtooth is a blockchain platform designed for enterprise usage. Sawtooth builds upon the learnings of platforms before it such as Bitcoin and Ethereum and is designed with a focus on modularity and extensability. Sawtooth is unique from existing blockchain platforms in that it allows developers to program their smart contracts in a variety of traditional programming languages. Future Sawtooth developments will focus on increasing the throughput and privacy of transactions.
```

Comment:
I'd say "lessons" instead of "learnings".

Reply:
Also, I would put commas before and after "such as Bitcoin and Ethereum".

Reply:
Alas, "unique from" is incorrect (according to us grammar pedants).  My first thought would be to say "distinct from" instead, but I like the the word "unique" here.  Perhaps change this sentence to "Sawtooth is unique among existing blockchain platforms ...".